docs/UserGuide.md

```diff
@@ -215,5 +227,5 @@ Action | Format, Examples
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
+**List** | `list l/KEYWORD` <br> e.g., `list people`, `list stats`
```

Comment:
I think that to match prefix, the e.g. would be sth like list l/people, list /stats. Please check on this.